prql-java/src/lib.rs

```diff
@@ -14,11 +14,23 @@ pub extern "system" fn Java_org_prql_prql4j_PrqlCompiler_toSql(
         .get_string(query)
         .expect("Couldn't get java string!")
         .into();
-    let rs_sql_str: String = prql_compiler::compile(&prql_query, &Options::default())
-        .expect("Couldn't compile query to prql!");
-    env.new_string(rs_sql_str)
-        .expect("Couldn't create java string!")
-        .into_raw()
+    let result = prql_compiler::compile(&prql_query, &Options::default());
+    return java_string_with_exception(result, &env);
```

Comment:
```suggestion
   java_string_with_exception(result, &env)
```

In rust we automatically return a final statement with no semicolon

Reply:
Yes, fixed.   Java test method for compile error added too.

```
    @Test(expected = Exception.class)
    public void compileWithError() throws Exception {
       PrqlCompiler.toSql("from table | filter id >> 1");
    }
```

Reply:
Great! There are a couple more that the linter found. When we fix those let's merge